modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-vsphere_{context}"]
+= Configuring registry storage for vSphere
+
+During installation, you must configure your registry to use storage.
+
+.Prerequisites
+
+* A cluster on vSphere.
+* PV with `ReadWriteMany` access mode.
+* Must have "100Gi" capacity.
+
+.Procedure
+
+. To configure your registry to use storage, change the `spec.storage.pvc` in the
+`configs.imageregistry/cluster` resource.
+
+. Provide a suitable persistent volume, such as Azure File or NFS.
```

Comment:
Would you use Azure or NFS with vSphere? I'd still move this info to the prereqs and show the commands to do these first two steps.